actors/builtin/miner/miner_actor.go

```diff
@@ -1204,6 +1207,7 @@ func confirmSectorProofsValid(rt Runtime, preCommits []*SectorPreCommitOnChainIn
 		// Unlock deposit for successful proofs, make it available for lock-up as initial pledge.
 		err = st.AddPreCommitDeposit(depositToUnlock.Neg())
 		builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to add pre-commit deposit %v", depositToUnlock.Neg())
+		rt.Log(rtt.DEBUG, "added precommit deposit, amt added: %v, new total: %v", depositToUnlock.Neg(), st.PreCommitDeposits)
```

Comment:
this (and the `require` above it) are actually unlocking deposit, not adding any